server/base/src/main/java/org/apache/accumulo/server/client/ClientServiceHandler.java

```diff
@@ -327,6 +336,14 @@ public Map<String,String> getTableConfiguration(TInfo tinfo, TCredentials creden
     return conf(credentials, config);
   }
 
+  @Override
+  public Map<String,String> getTableSpecificConfiguration(TInfo tinfo, TCredentials credentials,
+      String tableName) throws TException, ThriftTableOperationException {
+    return getTableConfiguration(tinfo, credentials, tableName).entrySet().stream()
+        .filter(entry -> Property.isValidTablePropertyKey(entry.getKey()))
+        .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
+  }
```

Comment:
I would have to re-run the unit tests and switch out the call but I was getting back something like 250 properties when not filtering vs maybe 160 when filtering so there were quite a few that came back as it was all properties. Property.isValidTablePropertyKey is called when processing the updates so any property that failed that check caused a failure. The other option is to filter out properties that are not valid but figured it would be better to be explicit about it and fail vs just silently filtering.